hamilton/lifecycle/default.py

```diff
@@ -563,11 +569,19 @@ def never_reached(wont_proceed: int) -> int:
 
         Note you can customize the error you want it to fail on and the sentinel value to use in place of a node's result if it fails.
 
+        For Parallelizable nodes, this adapter will attempt to iterate over the node outputs. If an error occurs, the sentinel value is
+        returned and no more iterations over the node will occur. If you set ``fail_all_parallel`` to be True, it only sends on sentinel
+        value into the parallelize sub-dag.
+
         :param error_to_catch: The error to catch
         :param sentinel_value: The sentinel value to use in place of a node's result if it fails
+        :param fail_all_parallel: Treat a Parallelizable as 1 failure (True) or allow the successful ones to go through (False).
+        :param sentinel_injection_tags: Node tag key:value pairs that allow sentinel injection
```

Comment:
Also, is there any reason you would want to set specific ones? Why not just have a fixed tag? Then you could create a simple decorator that calls out to it:

```python
@drop_errors(...)
```

(still not 100% sure how to use these so this might be wrong...)

Reply:
Attempted to make a decorator that creates a specific tag to allow sentinel injection.

[4308d0a](https://github.com/DAGWorks-Inc/hamilton/pull/1017/commits/4308d0a83c200609ae45c02efa6dc97ec147456a)

Reply:
Awesome, thanks! Would you mind adding a little documentation on that decorator with an example? I think its a little confusing what it does.